Packs/CommonPlaybooks/ReleaseNotes/2_4_14.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Extract Indicators From File - Generic v2
+
+- Fixed bug: extracting indicators from PDF files and image files
```

Comment:
Please mention that we don't rasterize PDF files anymore in this playbook, in case some users were relying on this playbook providing images of PDFs (I also don't think it's the job of this playbook anyway)